PLATFORM_COSTS.md

```diff
@@ -1,20 +1,19 @@
-## Platform Costs
+# Platform Costs
 
 Adding new Build variants uses up machine time in our [build farm](https://ci.adoptopenjdk.net/). There are two ways in which you can help!
 
-#### BYOD - Bring Your Own Device
+## BYOD - Bring Your Own Device
 
 We are able to accept hardware donations. Generally we try to share resources as best as possible so each machine may be used to build other variants as well.
 
-#### Use Existing Hardware
+## Use Existing Hardware
```

Comment:
I wonder if this sectoin could do an overhaul if we still want it. At the moment we don't even use many of these providers in the general build farm so this may be misleading people reading it.

Reply:
I Will update in a separate PR